dialog/guard.config.patch

```diff
@@ -4,11 +4,11 @@
     "path":"/hail/avali",
     "value":{  
       "default":[  
-        "Hi there stranger~",
-        "-Hums to themself-",
+        "Hi there, stranger!",
+        "*hums to themself*",
```

Comment:
Since this isn't a formally-defined format, I suppose it's up for debate. It'd be easier on me to not capitalize in that scenario, since that's already how I formatted it.